src/components/Property/Types/VectorProperty/ViewOptions/VectorDefaultView.tsx

```diff
@@ -10,19 +12,29 @@ interface Props {
   value: number[];
   additionalData: AdditionalDataVectorMatrix;
   isInt?: boolean;
+  labels?: string[];
 }
 
 export function VectorDefaultView({
   disabled,
   setPropertyValue,
   value,
   additionalData,
-  isInt = false
+  isInt = false,
+  labels = ['x', 'y', 'z', 'w']
 }: Props) {
   const { value: currentValue, setValue: setCurrentValue } =
     usePropListeningState<number[]>(value);
 
   const { MinimumValue: min, MaximumValue: max, SteppingValue: step } = additionalData;
+  // @TODO (2025-03-03, emmbr) This should be handled a better way... This is a bit of a
+  // hack and the max value is just arbitrarily chosen
+  const maxAllowedExtentForSlider = 1e12;
+  const shouldShowSlider = max.some((value, i) => {
+    // When no min/max is set, the marks for the slider cannot be nicely computed
+    const extent = value - min[i];
+    return isFinite(value - min[i]) && extent < maxAllowedExtentForSlider;
+  });
```

Comment:
Looks like this is copied from the numeric property

Should we put this in some type of utility function instead? So that, when we address it, we will have to deal with both places at once?